server/listeners/eventHandlers/prisonEvents/sentenceDatesChangedEventHandler.ts

```diff
@@ -20,24 +20,22 @@ export default class SentenceDatesChangedEventHandler {
     if (licence) {
       const prisoner = await this.prisonerService.getPrisonerDetail(nomisId)
 
-      await Promise.all([
```

Comment:
Turns out updating the same licence via two different endpoints concurrently causes some unexpected behaviour (i.e. status wasnt getting updated because the other endpoint was updating the same licence at the exact same time). Now calling these endpoints in sequence